cmake/std/atdm/README.md

```diff
@@ -1034,6 +1035,39 @@ $ bsub -x -Is -n 20 \
 ```
 
 
+### vortex
+
+Once logged on to 'vortex' (SRN), one can either build and configure on the
+login node or the compute node. Make sure to setup SSH keys as described in
+`/opt/VORTEX_INTRO` before trying to build on a compute node. To get a list of
+compute nodes, run `bhosts`, then simply ssh to a given hostname. For example to
+configur, build and run the tests for the default `cuda-debug` build for
```

Comment:
Change 'configur' to 'configure'.

Reply:
If this is running from a compute node, please show the commands to move on to a compute node to make this clear.

Reply:
I think we should wait to continue this work on this until #6724 is rejected or accepted. 

Currently, we run tests via raw `mpiexec` (you can do this by hopping on a login or compute node with ssh and running the `mpiexec` command or `ctest`). But, if we use the jsrun wrapper going forward, we will need to be within a compute node allocation from `bsub` before running tests.